panels/user-accounts/um-password-dialog.c

```diff
@@ -566,6 +625,18 @@ um_password_dialog_new (void)
 
         g_object_unref (builder);
 
+        um->cancellable = g_cancellable_new ();
+
+        g_dbus_proxy_new_for_bus (G_BUS_TYPE_SESSION,
+                                  G_DBUS_PROXY_FLAGS_NONE,
+                                  NULL,
+                                  "org.gnome.Shell.InputSourceManager",
+                                  "/org/gnome/Shell/InputSourceManager",
+                                  "org.gnome.Shell.InputSourceManager",
+                                  um->cancellable,
+                                  (GAsyncReadyCallback) input_source_manager_proxy_ready,
+                                  um);
```

Comment:
It's a bit dangerous that this is created asynchronously, because a caller can show the password dialog before it gets created, which would fail the call to set the password mode. I'd probably make this synchronous.